packaging/rpm/flightctl.spec

```diff
@@ -47,6 +47,9 @@ install -m 0755 packaging/greenboot/flightctl-agent-running-check.sh %{buildroot
 cp bin/flightctl-agent %{buildroot}/usr/bin
 cp packaging/must-gather/flightctl-must-gather %{buildroot}/usr/bin
 cp packaging/systemd/flightctl-agent.service %{buildroot}/usr/lib/systemd/system
+if [[ -d /etc/bash_completion.d ]]; then
+	bin/flightctl completion > /etc/bash_completion.d/flightctl-completion.sh
```

Comment:
Is this missing the argument of which shell to create the shell completion for?

Reply:
Not necessarily, Cobra can automatically detect the shell.